djstripe/models/base.py

```diff
@@ -109,6 +109,10 @@ def _api_create(cls, api_key=djstripe_settings.STRIPE_SECRET_KEY, **kwargs):
 		:type api_key: string
 		"""
 
+		for key in kwargs:
+			if isinstance(kwargs[key], StripeModel):
+				kwargs[key] = kwargs[key].id
+
```

Comment:
There's another way to handle this piece. Idk if it's still in the codebase, but check out how we handle passing plan ids to a subscription. I'm currently on mobile, but I can gather example links for you next time I get a free moment.

Reply:
Ah right, like this?

https://github.com/dj-stripe/dj-stripe/blob/master/djstripe/models/billing.py#L1039

Reply:
exactly